### DISCUSSION TITLE
Website Documentation Issue

### DISCUSSION THREAD

**Discussion by Brontomerus** [PrefectHQ/prefect#3879] [General]
I've noticed a minor update needed for the [Executor Documentation](https://docs.prefect.io/api/latest/executors.html#daskexecutor) on your website. Since that documentation had been written, I've noticed that dask-cloudprovider must have changed their package structure when they added support for other cloud platforms. Its nothing huge, just want to help because its one of those small details that could get lost in the shuffle of daily work. Below is the change needed to reflect what folks will need as it sits today:

```python
executor = DaskExecutor(
    cluster_class="dask_cloudprovider.FargateCluster",
    cluster_kwargs={
        "image": "prefecthq/prefect:latest",
        "n_workers": 5,
        ...
    },
)
```

And the change we'd need in there:
```python
# there is now a '.aws' in the import path
from dask_cloudprovider.aws import FargateCluster
```
so this would now be:
```python
from dask_cloudprovider.aws import FargateCluster
flow.executor = DaskExecutor(
    cluster_class="dask_cloudprovider.aws.FargateCluster",
    cluster_kwargs={
        "image": "prefecthq/prefect:latest",
        "n_workers": 5,
        ...
    }
)
```

**Comment by cicdw** [comment]
Hi @Brontomerus - would you like to open a PR with the suggested change?  Here is the line where you would need to add the `.aws.` submodule: https://github.com/PrefectHQ/prefect/blob/master/src/prefect/executors/dask.py#L111
